src/main/java/com/marklogic/contentpump/Command.java

```diff
@@ -670,6 +697,10 @@ public void applyConfigOptions(Configuration conf, CommandLine cmdline) {
                             + ": " + arg);
                 }
             }
+            // If api key is specified, ignore input from -ssl and enable ssl
+            if (conf.get(MarkLogicConstants.OUTPUT_API_KEY) != null) {
+                conf.set(MarkLogicConstants.OUTPUT_USE_SSL, "true");
```

Comment:
Why are the OUTPUT_USE_SSL and INPUT_USE_SSL being set as String "true" instead of conf.setBoolean() with `true`?

Reply:
Will fix it. Thanks!